packages/aws-rfdk/lib/deadline/lib/render-queue-ref.ts

```diff
@@ -377,6 +377,24 @@ export interface RenderQueueProps {
    * @default false
    */
   readonly enableLocalFileCaching?: boolean;
+
+  /**
+ * The credentials for configuring Deadline Secrets Management on the Render Queue. Providing this value will enable Secrets Management
+ * on the Render Queue.
+ *
+ * The secret must be in the following JSON format:
+ *
+ * ```
+ * {
+ *   username: string,
+ *   password: string,
+ * }
+ * ```
+ *
+ * @see https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html
+ * @default Deadline Secrets Management is not enabled
+ */
+  readonly secretsManagementCredentials?: ISecret;
```

Comment:
Sure. I had it this way to make enabling SM on the RQ an explicit step, but we can just update the docstrings so that it's clear that enabling SM on the Repository will implicitly enable it on the RQ. Updated.

I also updated our unit tests to disable SM wherever we used an RCS w/o TLS, so let's remember to add a "BREAKING CHANGE" section when we merge this feature branch in because farms that use the default `Repository` and a `RenderQueue` without TLS enabled won't `cdk synth` anymore.